modules/sfp_opencorporates.py

```diff
@@ -18,12 +18,25 @@ class sfp_opencorporates(SpiderFootPlugin):
     """OpenCorporates:Passive,Footprint,Investigate:Search Engines::Look up company information from OpenCorporates."""
 
     meta = {
-		'name': "OpenCorporates",
-		'summary': "Look up company information from OpenCorporates.",
-		'flags': [ "" ],
-		'useCases': [ "Passive", "Footprint", "Investigate" ],
-		'categories': [ "Search Engines" ]
-	}
+        'name': "OpenCorporates",
+        'summary': "Look up company information from OpenCorporates.",
+        'flags': [ "" ],
+        'useCases': [ "Passive", "Footprint", "Investigate" ],
+        'categories': [ "Search Engines" ],
+        'dataSource': {
+            'website': "https://opencorporates.com",
+            'model': "FREE_NOAUTH_UNLIMITED",
```

Comment:
I think it's FREE_NOAUTH_LIMITED since you need to upgrade to get higher query volumes.

Reply:
Updated